From 5622b32a764bcbe754aa12e18d9056a2d2001e49 Mon Sep 17 00:00:00 2001
From: Ed Watson <edmundwatson@gmail.com>
Date: Mon, 13 Oct 2025 13:32:35 +0200
Subject: [PATCH] Add aliases for mmc0 and mmc so sdcard and emmc don't change
 during boot.

Good checks
cat /proc/device-tree/aliases/mmc0
cat /proc/device-tree/aliases/mmc1
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index 88bafebef..ed56123d6 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -25,6 +25,8 @@ aliases {
 		serial1 = &usart6;
 		serial2 = &usart1;
 		ethernet0 = &eth1;
+		mmc0 = &sdmmc1;
+		mmc1 = &sdmmc2;
 	};
 
 	chosen {
